### ISSUE TITLE
Unable to run simple SSH connection via password

### ISSUE THREAD

**Issue by dwgnr** [th-nuernberg/usermgmt#83]
Currently, I can not do a simple `usermgmt list` with simple password authentication. 
The application provides the password prompt but seems to ignore it and continues with key-based authentication, then crashes: 

<img width="683" alt="image" src="https://github.com/th-nuernberg/usermgmt/assets/46081636/18b1d818-1f61-4cba-91b3-46020c46a87d">

It should be possible to use simple password auth without SSH agent and SSH keys provided.

**Comment by dwgnr** [comment]
Interestingly this seems to be an issue on Ubuntu 22 using the CLI compiled via cargo-deb.

**Comment by dwgnr** [comment]
Ok it seems the SSH auth is very broken on all systems. I noticed that I used an older version on my Mac. That's why I thought it was working. With the most recent version I can't establish any password-based authentication. When I remove the `ssh_key_path` variable from conf.toml it asks for the SSH password but proceeds with key-based auth:
<img width="664" alt="image" src="https://github.com/th-nuernberg/usermgmt/assets/46081636/03c97784-37ad-4885-8b24-c7029d5d2736">
